src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ CHANGELOG
  * Added `Argon2idPasswordEncoder`
  * Deprecated using `Argon2iPasswordEncoder` while only the `argon2id` algorithm
    is supported, use `Argon2idPasswordEncoder` instead
+ * Added two virtual methods (`__serialize` and `__unserialize`) to the `TokenInterface`
```

Comment:
Added methods `__serialize` and `__unserialize` to the `TokenInterface` ("virtual" is not needed imho - we describe the intent here, technically we follow our BC promise of course)